index.html

```diff
@@ -331,30 +331,32 @@
       drawSpecialLabel(ctx, radius, "");
       drawSchoolTime(ctx, radius, periodIndex);
       drawLunchButton(); // comment out for  all school lunch
+      drawFace(ctx, radius/5, radius/2.5, radius/3);
+      drawSchoolDay(ctx, radius);
   }
 }
 
-function drawFace(ctx, radius) {
+function drawFace(ctx, radius, x, y) {

```

Comment:
I know there wasn't one here before, but it can't hurt to add a comment describing what the parameters do. In particular, do x and y refer to the center of the face, or the top left corner, or some other point?